Marlin/src/module/stepper_indirection.cpp

```diff
@@ -181,7 +181,7 @@
     st.power_down_delay(128); // ~2s until driver lowers to hold current
     st.hysterisis_start(0); // HSTRT = 1
     st.hysterisis_low(1); // HEND = -2
-    st.diag1_active_high(1); // For sensorless homing
+    st.diag1_active_high(0); // For sensorless homing
```

Comment:
Fixes a bug?

Reply:
It was advised by UltiMachine because the diag pins are connected. I didn't ask further questions and for sensorless homing it doesn't really matter as the triggering logic can be inverted in the configuration files.

Reply:
How will this affect boards other than Archim?

Reply:
The user will need to change the `_ENDSTOP_INVERTING` setting. No change is needed if sensorless homing isn't used.

Reply:
Seems sensible!